src/components/Form/Widgets/FileWidget.tsx

```diff
@@ -204,9 +207,11 @@ export const FileWidget = ({
 							sx={{ fontSize: 64, color: (theme) => theme.palette.grey[500] }}
 						/>
 					)}
-					<Typography ml={2}>Drag and drop or browse files</Typography>
-				</Box>
-			</Box>
+					<Typography ml={2}>
```

Comment:
remove `ml={2}` for `gap={2}` on the Stack

Reply:
Are you doing it or do you want me to push :P ?